src/content/docs/infrastructure/infrastructure-integrations/get-started/understand-use-data-infrastructure-integrations.mdx

```diff
@@ -31,35 +31,155 @@ redirects:
   - /docs/infrastructure/amazon-integrations/find-use-data/understand-use-integration-data
   - /docs/infrastructure/microsoft-azure-integrations/find-use-data/understand-use-integration-data
   - /docs/infrastructure/integrations-sdk/understand-use-data/understand-use-integration-data
+  - /docs/infrastructure/infrastructure-integrations/get-started/use-integration-data-new-relic-dashboards
+  - /docs/integrations/infrastructure-integrations/get-started/use-integration-data-new-relic-dashboards
+  - /docs/infrastructure/integrations-getting-started/getting-started/use-integration-data-new-relic-insights
+  - /docs/integrations/getting-started/getting-started/use-integration-data-new-relic-insights
+  - /docs/integrations/new-relic-integrations/getting-started/use-integration-data-new-relic-insights
+  - /docs/integrations/infrastructure-integrations/get-started/use-integration-data-new-relic-insights-dashboards
 ---
 
-With our infrastructure integrations, you can monitor the performance of many popular services. Our infrastructure integrations are separated into two main categories:
-
-* [Cloud integrations](/docs/integrations/new-relic-integrations/getting-started/introduction-infrastructure-integrations#cloud): Integrations for cloud platform services, including AWS, Azure, and GCP.
-* [On-host integrations](/docs/integrations/new-relic-integrations/getting-started/introduction-infrastructure-integrations#on-host): "On-host" refers to core services integrations that you can install directly on a host. Examples: MySQL, NGINX, Kubernetes, Redis.
-
-Here are some tips on how to find, understand, and use data reported from infrastructure integrations.
+With our infrastructure integrations, you can monitor the performance of many popular services. Here are some tips on how to find, understand, and use data reported from infrastructure integrations.
 
 ## Explore your infrastructure integration's data [#explore]
 
 The best way to understand infrastructure integrations's data and see what you can do with it is to enable an integration and [explore the data](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) in the New Relic UI.
```

Comment:
This sentence is outside your scope, but could you update this? This should be more like this: "Install the infrastructure agent so you can see its data in the UI."